package.json

```
@@ -2,7 +2,7 @@
 	"name": "vscode-wing-cf-tools",
 	"displayName": "Cloud Foundry Tools",
 	"description": "Vscode extension of Cloud Foundry tools",
-	"version": "5.6.3",
+	"version": "5.6.2",
```

```suggestion
	"version": "5.6.4",
```
it is impossible to release a previous version :)